spiped/spiped.1

```diff
@@ -120,7 +120,11 @@ Disable transport layer keep-alives.
 .TP
 .B \-n <max # connections>
 Limit on the number of simultaneous connections allowed.
-This value must be between 1 and 500.
+A value of 0 indicates that no limit should be imposed; this may be
```

Comment:
Given the (astonishing) difficulty of parsing `-n 0`, and the fact that we don't actually make this unlimited (as it's still bound by `SIZE_MAX`), why don't we remove this "value of 0" stuff?  If somebody wants a (virtually) unlimited number of connections, they can do `-n 1234567890`.  I doubt that anybody will have more than 2^30 connections.